### DISCUSSION TITLE
Restructuring builds

### DISCUSSION THREAD

**Discussion by pjb7687** [gnudatalanguage/gdl#983] [General]
Currently, we have two types of build on Linux/macOS: `full` and `mini`
And on Windows: `full-msmpi`, `full`, and `mini`

I think we can restructure this.

Firstly I think the name `mini` is not intuitive. From the previous discussion (https://github.com/gnudatalanguage/gdl/pull/902), I was able to understand the main purpose to have `mini` build is for headless HPCs (since it targets HPCs, it makes sense to enable MPI with this).

But then, I think that it is too much to disable everything with `mini` builds. For example, plplot can work without GUI, and this can be used to plot to files... In principle, I think we can keep everything without wxWidgets?

Also, I suggest changing the name `mini` to something more intuitive, more clear it is targeting HPCs. How about `headless`? Also rather than `full`, I would say something like `standard`. I am open to any other name suggestions too.

My suggestion:
1. `headless`: we just get rid of wxWidgets, and keep everything else + MPI
2. `standard`: everything without MPI

**Comment by slayoo** [comment]
related: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=753618
(wishlist item for a gdl-nox pacage)
